### DISCUSSION TITLE
Daily Leaderboard Scroll to bottom

### DISCUSSION THREAD

**Discussion by Eothred** [monkeytypegame/monkeytype#4013] [Ideas] [answered]
I am one of those that barely make it to the daily leaderboards sometimes. I am hence often curious what's the 1000th place whenever I am not qualified. Currently I need to scroll a lot to find that (hence only did it once). I was pondering about suggesting if e.g. the button to get to my spot in the list could send you to the bottom of the daily leaderboard when a user is currently not qualified to be on the list? Would that make sense UI-wise?

I think many "not qualified" users might be curious how far off they are from making the list.

**Comment by WhisperinCheetah** [comment, marked as answer]
They have added a similar feature where you can see what speed you need to qualify, given that you have done a run in that category that day.

![image](https://github.com/monkeytypegame/monkeytype/assets/47121778/304dac1a-6dec-4618-a7b8-68a27e6bc86a)

**Reply by Eothred** [reply]
Thanks for pointing that out! I think that exactly solves the use case I was considering.
